site/content/en/docs/start/_index.md

```diff
@@ -150,9 +150,11 @@ In another window, start the tunnel to create a routable IP for the 'balanced' d
 minikube tunnel
 ```
 
-To find the routable IP, run this command and examine the `EXTERNAL-IP` column:
+To find the routable external IP, run this command:
 
-`kubectl get services balanced`
+```shell
+kubectl get services balanced --output=jsonpath='{.status.loadBalancer.ingress[0].ip}''
```

Comment:
I would rather not: I really do not expect folks to be able to memorize this command.

Reply:
sounds reasonable. will close